client/src/pages/Workspace/index.tsx

```diff
@@ -51,20 +51,18 @@ function WorkspacePage() {
         </SelectModal>
       )}
 
-      {MODAL_MENUS.map(({ id, props: { title, texts, btnText } }) => {
-        if (id === clickedMenuId)
-          return (
-            <WorkspaceModal
-              key={id}
-              {...{ title, texts, btnText }}
-              inputValue={inputValue}
-              onChange={onInput}
-              onClose={() => setClickedMenuId(0)}
-              onClick={onClickBtn}
-              isInputDisabled={clickedMenuId === MENU.JOIN_SUCCESS_ID}
-            />
-          );
-      })}
+      {clickedMenuId !== 0 && (
```

Comment:
`!== 0`이라면 이렇게 수정하는건 어떤가요? 
혹시 의미를 확실히 하기 위함이라면 수정하지 않아도 괜찮을것 같아요!

```suggestion
      {clickedMenuId && (
```

Reply:
저는 `!== 0`하는게 바로 보여서 좋아요 ㅎㅎ

Reply:
코드를 보니까 `0`이 특별한 의미를 가지는 것 같더라고요. 개인적으로 `!== 0`를 더 선호하기도 하고요.